.github/workflows/connector-tests.yml

```
@@ -0,0 +1,130 @@
+        continue-on-error: true
+      # Get the monorepo so we can test the connectors
+      - name: Checkout Airbyte Monorepo
+        uses: actions/checkout@v4
+        if: steps.no_changes.outcome != 'failure'
+        with:
+          repository: airbytehq/airbyte
+          ref: master
+      - name: Fetch last commit id from remote branch [PULL REQUESTS]
+        if: github.event_name == 'pull_request' && steps.no_changes.outcome != 'failure'
+        id: fetch_last_commit_id_pr
+        run: echo "commit_id=$(git ls-remote --heads origin refs/heads/${{ github.head_ref }} | cut -f 1)" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:98: Double quote to prevent globbing and word splitting

(shellcheck)